tests/test_config.py

```diff
@@ -6,19 +6,6 @@
 
 
 class TestGenerateConfig(unittest.TestCase):
-    @patch('config.ConfigParser')
```

Comment:
Is it possible to retain some version of these tests?

Reply:
the last commit re-adds the tests and ensures they are passing